web/docs/guides/auth/auth-twitch.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: auth-twitch
+title: "Setting up Twich Auth"
+description: Add Twitch OAuth to your Supabase project
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+To enable Twitch Auth for your project, you need to set up a Twitch Application and add the Application OAuth credentials to your Supabase Dashboard.
+
+## Overview
+
+Setting up Twitch logins for your application consists of 3 parts:
+
+* Create and configure a Twitch Application [Twitch Developer Console](https://dev.twitch.tv/console)
+* Add your Twitch OAuth Consumer keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Steps
+
+### Access your Twitch Developer account
+
+- Go to [dev.twitch.tv](https://dev.twitch.tv).
+- Click on `Log in with Twitch` at the top right to log in.
+- If you have not already enabled 2-Factor Authentication for your Twitch Account, you will need to do that at [Twitch Security Settings](https://www.twitch.tv/settings/security) before you can continue.
+
+![Twitch Developer Page](/img/guides/twitch-developer-page.png)
+
+- Once logged in, go to the [Twitch Developer Console](https://dev.twitch.tv/console).
+
+![Twitch Developer Console](/img/guides/twitch-console.png)
+
+### Find your callback URL
+
+In the next step you require a callback URL, which looks like this: 
+
+`https://<project-ref>.supabase.co/auth/v1/callback`
+
+- Go to your [Supabase Project Dashboard](https://supabase.io).
+- Click on the `Settings` icon at the bottom of the left sidebar.
+- Click on `API` in the list.
+- Under Config / URL you'll find your API URL, you can click `Copy` to copy it to the clipboard.
+- Now just add `/auth/v1/callback` to the end of that to get your full `OAuth Redirect URI`.
+
+<video width="99%" muted playsInline controls="true">
+<source src="/videos/api/api-url-and-key.mp4" type="video/mp4" muted playsInline />
+</video>
+
+
+### Create a Twitch Application
+
+- Click on `+ Register Your Application` at the top right.
+
+![Register Application](/img/guides/twitch-register-your-application.png)
```

Comment:
perhaps move this one below the next bullet point?  I read `+ Register Your Application` on the bullet above, then looked at this screenshot and tried to find the button. I realised then that the image is related to the next 2 bullet points.